### PR TITLE
Add a tutorial for creating an OpenVINO detector

### PR DESCRIPTION
**Pull request by velovix**
This tutorial guides the user through creating an object detection capsule from a pre-trained OpenVINO model. Also, this re-organizes some of our existing encapsulation docs to be under a separate section.

The document on creating a classifier will be part of a separate PR.

### CODE REVIEW

File: docs/creating_capsules/openvino_object_detector.rst [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+of the capsule directory and the ``description`` field is a short,
+human-readable description of the capsule's purpose. The other fields are a bit
+more complex, so let's break each one down. For more information on each of
+these fields, check out the `The Capsule Class`_ reference documentation.
+
+.. code-block:: python
+
+   version = 1
+
+This is the capsule's version (not to be confused with the version of the
```
Comment (BryceBeagle) [inline docs/creating_capsules/openvino_object_detector.rst:87]: Maybe explain that it does not follow semver

File: docs/creating_capsules/openvino_object_detector.rst [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+.. code-block:: python
+
+   backend_loader = lambda capsule_files, device: Backend(
+       model_xml=capsule_files["face-detection-adas-0001.xml"],
+       weights_bin=capsule_files["face-detection-adas-0001.bin"],
+       device_name=device
+   )
+
+Here we define a lambda function that creates an instance of a Backend class
+with the model and weights file, as well as the device this backend will run
```
Comment (BryceBeagle) [inline docs/creating_capsules/openvino_object_detector.rst:124]: file->files

File: docs/creating_capsules/openvino_object_detector.rst [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+on. We will define this Backend class in the next section.
+
+.. code-block:: python
+
+   options = common_detector_options
+
+We give this capsule some basic options that are common among most all
```
Comment (BryceBeagle) [inline docs/creating_capsules/openvino_object_detector.rst:131]: `most all` -> `most`

File: docs/creating_capsules/openvino_object_detector.rst [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+expects. Also provided is a resize object, which contains the necessary
+information to map the resulting detections to the coordinate system of the
+originally sized video frame.
+
+.. code-block:: python
+
+   prediction = self.send_to_batch(input_dict).get()
+
+Next, the input data is sent into the model for batch processing. We
+immediately start blocking until the result is ready with ``get``. The results
```
Comment (BryceBeagle) [inline docs/creating_capsules/openvino_object_detector.rst:204]: Suggestion:
`We immediately start blocking` -> `The capsule will block and wait until`

File: docs/creating_capsules/packaging_a_capsule.rst [hunk 5]
```diff
@@ -0,0 +1,15 @@
+###################
+Packaging a Capsule
+###################
+
+For capsule and application developers, OpenVisionCapsules provides a function
+to package up unpackaged capsules. The optional ``key`` field encrypts the
+capsule with AES.
```
Comment (BryceBeagle) [inline docs/creating_capsules/packaging_a_capsule.rst:7]: Do we need to mention the key size? i.e. # of bits?

**Comment by velovix** [comment]
Never mind, this is going in the BrainFrame repository instead!

**Pull request closed without merging**